klarna-on-demand/build.gradle

```diff
@@ -29,15 +17,24 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
-}
 
-dependencies {
-    androidTestCompile ('org.powermock:powermock-easymock-release-full:1.5.6') {
-        exclude group : 'org.hamcrest'
-        exclude module: 'objenesis'
+    sourceSets {
+        androidTest.setRoot('src/test')
     }
+}
 
-    androidTestCompile 'org.easymock:easymock:3.3'
+apply plugin: 'android-unit-test'
```

Comment:
With all these changes in the gradle, it would be prudent to check if the [publishing instructions](https://confluence.internal.machines/display/teamindex/Deploying+to+Maven) still work.